lnrpc/walletrpc/walletkit_server.go

```diff
@@ -1666,7 +1673,9 @@ func parseAddrType(addrType AddressType,
 	switch addrType {
 	case AddressType_UNKNOWN:
 		if required {
-			return nil, errors.New("an address type must be specified")
+			return nil, errors.New(
```

Comment:
nit: for `errors` we'd format it similar to `fmt.Errorf`.

Reply:
Thanks I missed this style fix. Will add.